scrape.py

```diff
@@ -42,6 +42,20 @@
 
 # All the files we want to list
 files = [
+    # B2GDroid
```

Comment:
The mock up in [Bug 1213905](https://bugzilla.mozilla.org/show_bug.cgi?id=1213905) shows B2GDroid on top. Maybe we can rearrange the position when campaign(?) ends?